lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -6,6 +6,7 @@ use 5.010;
 use base qw( SyTest::Homeserver );
 
 use Carp;
+use Socket qw( pack_sockaddr_un );
```

Comment:
Is this from a CPAN module? I can't see to find it when using the docker container.